eppo/src/main/java/cloud/eppo/android/EppoClient.java

```diff
@@ -140,6 +141,11 @@ public Builder assignmentLogger(AssignmentLogger assignmentLogger) {
       return this;
     }
 
+    public Builder ignoreCache(boolean ignoreCache) {
```

Comment:
we have many caches going on in this SDK; let's make this more explicitly named. Perhaps something like `ignoreCachedConfiguration()`